src/main/java/com/minecolonies/coremod/tileentities/ScarecrowTileEntity.java

```diff
@@ -1,9 +1,10 @@
 package com.minecolonies.coremod.tileentities;
 
+import com.minecolonies.api.util.EntityUtils;
 import com.minecolonies.coremod.colony.Colony;
 import com.minecolonies.coremod.colony.ColonyManager;
 import com.minecolonies.coremod.inventory.InventoryField;
-import com.minecolonies.coremod.util.EntityUtils;
+import com.minecolonies.coremod.lib.Constants;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.lib.Constants'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)